src/services/api/resources/DatasetsAPI.js

```diff
@@ -11,9 +11,9 @@ export default class DatasetsAPI extends DatagouvfrAPI {
    * @param {str?} sort
    * @returns {object}
    */
-  async getDatasetsForOrganization(org_id, page = 1, sort = '-created') {
+  async getDatasetsForOrganization(orgId, page = 1, sort = '-created') {
     // WARNING: specify `-created` or another sort explicitely because default sort has a pagination issue
-    const url = `${this.url()}/?organization=${org_id}&page=${page}&sort=${sort}&page_size=21`
+    const url = `${this.url()}/search/?organization=${orgId}&page=${page}&sort=${sort}&page_size=21`
```

Comment:
> Fix la page détail d'une organisation en utilisant le bon endpoint sur l'API v2.

@abulte tu mentionnes v2 mais `this.url()` est tjs en v1 ici, non ?

Reply:
Depends comment la classe est instanciée, et dans notre cas c'est en v2 (a bit touchy).

Reply:
Ouch ok j'avais oublié les 2 instances différentes. Pas évident à suivre en effet. Mais rien à changer ici.

Reply:
On verra si on peut améliorer la situation avec https://github.com/ecolabdata/ecospheres-front/issues/271